packages/compiler/core/node-host.ts

```diff
@@ -36,7 +36,7 @@ export const NodeHost: CompilerHost = {
   },
   getSourceFileKind: getSourceFileKindFromExt,
   logSink: createConsoleSink(),
-  mkdirp: (path: string) => mkdirp(path),
+  mkdirp: (path: string) => mkdirp(path) as any,
```

Comment:
What happened here?

Reply:
the typing for that library is returning `Promise<undefined | void>` (void) causing issue. this triggered me also to create that remove mkdirp library issue as it is useless now